### DISCUSSION TITLE
Migration von Homegear nach Raspberrymatic ohne Neuanlernen durch Schlüsselübertragung möglich?

### DISCUSSION THREAD

**Discussion by lhurt** [OpenCCU/OpenCCU#2195] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,
ich würde gerne von Homegear nach Raspberrymatic wechseln. Dabei würde ich gerne vermeiden meine 40 Geräte ab- und wieder anzumelden. Teilweise sind die Geräte etwas verbaut. Daher habe ich mir überlegt den AES Key, der bei Homematic durch den User konfiguriert werden kann einfach zu übernehmen.

Kann mir bitte jemand einen Tipp geben, ob das überhaupt funktioniert, oder ob Probleme mit dem Vorgehen vorhersehbar sind?

Mein System soll auf einem Raspberry3 mit `HM-MOD-RPI-PCB` auf Docker laufen.

Vielen Dank schon mal.

**Comment by jens-maus** [comment, marked as answer]
Tut mir leid, aber ein solcher kompletter Systemwechsel von Homegear zu RaspberryMatic kann meines Wissen nur mit einem kompletten neu anlernen aller Geräte passieren. Auch wenn man vielleicht die verschlüsselte Kommunikation zu/zwischen den Homematic Geräten mit übernehmen des AES Keys noch hinbekommen könnte, so fehlen die Geräte ja dann trotzdem noch mit all Ihren Einstellungen, etc. in der zugrundeliegenden Gerätedatenbank in RaspberryMatic. Und aktuell ist mir kein Weg bekannt wie man ein Gerät oder eine ganze Homegear Konfiguration zu RaspberryMatic übernehmen könnte samt der Geräteinformationen, etc.
